Code.gs

```diff
@@ -19,11 +19,11 @@
 var targetCalendarName = "";           // The name of the Google Calendar you want to add events to
 var sourceCalendarURL = "";            // The ics/ical url that you want to get events from
 
-var howFrequent = 15;                  // What interval (minutes) to run this script on to check for new events
+var howFrequent = 30;                  // What interval (minutes) to run this script on to check for new events
```

Comment:
Please don't change default values (such as `howFrequent` or `addAlerts` unless that is the specific issue)